src/AutoRest.CSharp/Common/Output/Models/Types/SerializableObjectType.cs

```diff
@@ -22,6 +23,8 @@ protected SerializableObjectType(string defaultNamespace, SourceInputModel? sour
         {
         }
 
+        public INamedTypeSymbol? GetExistingType() => ExistingType;
```

Comment:
why we need this method wehn we have the public property `ExistingType`?

Reply:
`ExistingType` is protected. This method will be become unused when serialization method creation is moved into `ModelTypeProvider`, which is done in consolidation PR.